src/Sylius/Bundle/AdminBundle/EventListener/ResourceDeleteSubscriber.php

```diff
@@ -118,4 +122,13 @@ private function isAdminSection(array $syliusParameters): bool
     {
         return array_key_exists('section', $syliusParameters) && 'admin' === $syliusParameters['section'];
     }
+
+    private function getFlashBag(): FlashBagInterface
```

Comment:
Maybe it could be cool to add this on an Internal trait to simplify removing the bc-layer later.